lib/api/stitch_plan.py

```diff
@@ -17,6 +17,7 @@ def get_stitch_plan():
         return dict(colors=[], stitch_blocks=[], commands=[])
 
     metadata = g.extension.get_inkstitch_metadata()
+
```

Comment:
Let's roll back this change, no need for a blank line here now.

Reply:
Removed in latest commit.